web/src/compose.js

```diff
@@ -67,7 +67,8 @@ export function compute_show_video_chat_button() {
 
     if (
         page_params.realm_video_chat_provider === available_providers.jitsi_meet.id &&
-        !page_params.jitsi_server_url
+        !page_params.jitsi_server_url &&
+        !page_params.realm_jitsi_server_url
```

Comment:
It's fairly awkward that we need to check both of these values.. but it is making me think we have the API not quite right.